asset/asset.json

```diff
@@ -1,4 +1,4 @@
 {
     "name": "elasticsearch",
-    "version": "3.5.2"
+    "version": "3.5.3"
```

Comment:
Is the version bump necessary?  Does it do anything since all we change is docs?

Reply:
I think you're right, we wouldn't make a new release.  So I'll change that back.